docs/contrib/Logger.rst

```diff
@@ -1,29 +1,94 @@
 Using the logging infrastructure
 ================================
 
-In order to ensure consistent messaging from the openroad application we
-have adopted spdlog as our logging infrastructure. We have a thin
-wrapper on top for extensibility. Whenever a message needs to be issued
-you will use one of the logging functions in the ‘ord’ namespace.
+OpenROAD uses spdlog as part of logging infrastructure in order to ensure a clear, consistent messaging and complete messaging interface. 
+A wrapper formats the prefix in the reccommended messaging style and limit. 
+A message format is as follows:
+<tool id> <message id> : <Message body>.
+
+For Example,
+[INFO ODB-0127] Reading DEF file: ./results/asap7/aes/base/4_cts.def
 
 All output from OpenROAD tools should be directed through the logging
-API so that redirection, file logging and execution control flow is
-handled consistently.
+API to ensure that redirection, file logging and execution control flow is
+handled consistently. This also includes messages from a third party tool integration.
+Use one of the logging functions in the ‘ord’ namespace to issue the message.
```

Comment:
The quote around ord looks to be unicode, is better to use "base" characters `'` or `"` or `

Reply:
The messages come from a "third party tool", not a "third party tool integration".
There are no "logging functions in the ‘ord’ namespace".
It should say "Use the 'ord' message ID for third party tools".